packages/suite-desktop-api/src/messages.ts

```diff
@@ -0,0 +1,23 @@
+import { ExtractUndefined } from './methods';
+
+export type SuiteThemeVariant = 'light' | 'dark' | 'system';
+
+export type UpdateInfo = {
```

Comment:
nit: Prefer using `interface` over `type` >>> https://ncjamieson.com/prefer-interfaces/

Reply:
`SuiteThemeVariant` is a param of `DesktopApi.setTheme` method and contains values expected by `Electron.NativeTheme.themeSource`

in current develop it's typed twice, as `SuiteThemeVariant` in [global.d.ts](https://github.com/trezor/trezor-suite/blob/6fe89b1a2bb9e0a06278c2ba34d5bd5d78e03319/packages/suite/global.d.ts#L1) and `SuiteThemeVariantOptions` is [suite-types](https://github.com/trezor/trezor-suite/blob/6fe89b1a2bb9e0a06278c2ba34d5bd5d78e03319/packages/suite/src/types/suite/index.ts#L136)

Reply:
Thanks for explanation now it makes more sense now.